cmake/extension.cmake

```diff
@@ -138,6 +138,15 @@ endif()
 if(DA MATCHES ecdsa256)
   client_sdk_compile_definitions(
     -DECDSA256_DA)
+  if(DA STREQUAL tpm20_ecdsa256)
```

Comment:
can u move this after line 160
`elseif(DA MATCHES epid)`
`  message(WARNING "EPID support is no longer available")	  message(WARNING "EPID support is no longer available")`
`endif()`	
`endif()`

so that we can remove too many conditional nesting. faced a few issues with older makefile. it will start to get really hard to figure out the branches.

Reply:
It would have to be a separate 'if', and not an 'elseif'. This is because condition 'DA MATCHES ecdsa256' is also true for the condition 'DA STREQUAL tpm20_ecdsa256'. Should I go ahead with it?

Reply:
yes . This is a separate "if" condition not really dependent on the first condition. Even though it matches with ecdsa256. i would prefer to move it out.  as a simple "if DA STREQUAL tpm20_ecdsa256"

Reply:
Done.